test/spec/modules/adtargetBidAdapter_spec.js

```diff
@@ -0,0 +1,338 @@
+import { expect } from 'chai';
+import { spec } from 'modules/adtargetBidAdapter.js';
+import { newBidder } from 'src/adapters/bidderFactory.js';
+import { config } from 'src/config.js';
+
+const DISPLAY_REQUEST = {
+  'bidder': 'adtarget',
+  'params': {
+    'aid': 12345
+  },
+  'schain': { ver: 1 },
+  'userId': { criteo: 2 },
+  'mediaTypes': { 'banner': { 'sizes': [300, 250] } },
+  'bidderRequestId': '7101db09af0db2',
+  'auctionId': '2e41f65424c87c',
+  'adUnitCode': 'adunit-code',
+  'bidId': '84ab500420319d',
+};
+
+const VIDEO_REQUEST = {
+  'bidder': 'adtarget',
+  'mediaTypes': {
+    'video': {
+      'playerSize': [[480, 360], [640, 480]]
+    }
+  },
+  'params': {
+    'aid': 12345
+  },
+  'bidderRequestId': '7101db09af0db2',
+  'auctionId': '2e41f65424c87c',
+  'adUnitCode': 'adunit-code',
+  'bidId': '84ab500420319d'
+};
+
+const SERVER_VIDEO_RESPONSE = {
+  'source': { 'aid': 12345, 'pubId': 54321 },
+  'bids': [{
+    'vastUrl': 'http://rtb.adtarget.com/vast/?adid=44F2AEB9BFC881B3',
```

Comment:
Can you convert this url to `https`, as since Prebid 3.0 we restrict making calls to not secure external Urls.

Reply:
sure